tools-local/tasks/mobile.tasks/WasmBundleTask/TimeZoneLoader.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net;
+using System.Reflection;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class TimeZoneLoader : Task
+{
+    public string? InputDirectory { get; set; }
+    public string? OutputDirectory { get; set; }
+    public string? Version { get; set; }
+
+    private void DownloadTimeZoneData() {
+        List<string> files = new List<string>() {"africa", "antarctica", "asia", "australasia", "etcetera", "europe", "northamerica", "southamerica", "zone1970.tab"};
+        using (var client = new WebClient())
+        {
+            foreach (var file in files) {
+                client.DownloadFile($"https://data.iana.org/time-zones/tzdb-{Version}/{file}", $"{InputDirectory}/{file}");
+            }
+        }
+
+        files.Remove("zone1970.tab");        
+
+        using (Process process = new Process()) {
+            process.StartInfo.UseShellExecute = false;
+            process.StartInfo.FileName = "zic";
+            foreach (var f in files) {
+                process.StartInfo.Arguments = $"-d \"{OutputDirectory}\" \"{InputDirectory}/{f}\"";
+                process.Start();
+                process.WaitForExit();
+            }
+        }
+        File.Copy(Path.Combine(InputDirectory!,"zone1970.tab"), Path.Combine(OutputDirectory!,"zone1970.tab"));
+    }
+
+    private void FilterTimeZoneData(string[] areas) {
+        var directoryInfo = new DirectoryInfo (OutputDirectory!);
+        foreach (var entry in directoryInfo.EnumerateDirectories()) {
```

Comment:
Can we use `foreach (var info in Directory.EnumerateDirectories(OutputDirectory))` here?